Source/m_menu.c

```diff
@@ -3016,7 +3016,6 @@ static const char *weapon_attack_alignment_strings[] = {
 
 setup_menu_t weap_settings1[] =  // Weapons Settings screen       
 {
-  {"ENABLE RECOIL", S_YESNO,m_null,WP_X, WP_Y+ weap_recoil*8, {"weapon_recoil"}},
```

Comment:
So, we are really kicking this feature out of the menu?

Reply:
This demo is incompatible and I doubt anyone will complain. Crispy recoil is much better.

Reply:
But demos recorded with this feature enabled (would) still play back correctly, right?

Reply:
Right. I guess we can make both available, but one affects gameplay and the other is purely cosmetic.

Reply:
I think I'd prefer that. It's one of the original MBF fatures after all.

Didn't we want to rearrange that part of the menu anyway? 😉